lib/wordlist.js

```diff
@@ -14683,7 +14755,6 @@ module.exports = [
 'dilative',
 'dilatorily',
 'dilatory',
-'dildo',
 'dildoe',
```

Comment:
could you maybe replace this too while you're at it? thanks

Reply:
Done!